src/Templater.js

```diff
@@ -20,16 +20,57 @@ export default class Templater {
 
         this._built = false;
 
-        this._i18n = config.i18n;
+        this._mapper = config.mapper || config.i18n || this._mapper();
 
         this._trans = this._trans.bind(this);
     }
 
 
+    _mapper () {
+        return {
+            trans (lang, key, variables) {
+
+                if (typeof variables === 'undefined') {
+
+                    // support for trans(key, variable)
+                    if (typeof key === 'object') {
+                        variables = key;
+                        key = lang;
+                    }
+
+                    // support for trans(key)
+                    else if (typeof key === 'undefined') {
+                        key = lang;
+                    }
+
+                    // support for trans(lang, key)
+                    else {
+                        variables = {};
+                    }
+                }
+
+
+                let str = key;
+
+                /**
+                 * Replace variables in the string
+                 * `Hello :name!` => `Hello John!`
+                 * `:name aloha!` => `John aloha!`
+                 */
+                _(variables).forOwn((value, _key) => {
+                    str = str.replace(new RegExp(`:${_key}`, 'g'), value);
```

Comment:
This will incorrectly handle other variables like 
```js
> str = 'This is big :potato_master, but :potato is better'
'This is big :potato_master, but :potato is better'

> str.replace(/:potato_master/g, 'huehue')
'This is big huehue, but :potato is better'
// looks fine

> str.replace(/:potato/g, 'huehue')
'This is big huehue_master, but huehue is better'
// err.. :potato_master shouldn't have been replaced
```

Reply:
ill add `\b`?